### DISCUSSION TITLE
Instead of Gsuite can we give smtp credentials to send mails ?

### DISCUSSION THREAD

**Discussion by csetarun2** [Netflix/dispatch#2693] [Q&A] [answered]
Im setting up the mail integration, whenever incident created it will send to mail to user, I could see gsuite asking all info, instead of giving all details, can we give some smtp server details, there it will send details to users, will it be possible ?

**Comment by mvilanova** [comment, marked as answer]
@csetarun2 are you asking about support for allowing to use any SMTP server? If so, that would require creating a new email plugin. We can look into it in the future, but it's currently not a priority for us.

**Reply by csetarun2** [reply]
Thanks @mvilanova  , ya that would be great, instead  of giving the gsuite credentials, if we can give smtp server that would be easy
